neofetch

```diff
@@ -2686,7 +2686,8 @@ get_image_size() {
 make_thumbnail() {
     # Name the thumbnail using variables so we can
     # use it later.
-    image_name="$crop_mode-$crop_offset-$width-$height-${image//'/'/_}"
+    # fix in mac cannot replace '/' with '_', optionally can use ${image//\//_}
+    image_name="$crop_mode-$crop_offset-$width-$height-${image##*/}"
```

Comment:
Because originally we used `${image//'/'/_}`, I think it's better to change it to `${image//\//_}` instead.

Aside from that, LGTM.

Reply:
You are right. I have considered that. I am just trying to avoid generating very long filename (just in the case you wish to use images from any long filepath. And becasue there is no string checking, short filename could avoid potential issues from specific situations).